learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit/integrate-monerium-and-safe.md

```diff
@@ -0,0 +1,184 @@
+# How to integrate Monerium with your Safe
+
+The [`MoneriumPack`](https://github.com/safe-global/safe-core-sdk/tree/main/packages/onramp-kit/src/packs/monerium) enables Safe users to make direct transfers of digital money from their Safe addresses to an IBAN via the SEPA network. This allows them to use Monerium and Safe services together.
+
+More info about Monerium:
+
+- [Monerium](https://monerium.com)
+- [Monerium Developers](https://monerium.dev)
+
+### What are we going to learn?
+
+This guide demonstrates how to use the `MoneriumPack` with the [`OnRampKit`](https://github.com/safe-global/safe-core-sdk/tree/main/packages/onramp-kit) and incorporate it into your web application.
+
+### Prerequisites
+
+1. [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+2. [Monerium account and application](https://monerium.dev/docs/getting-started/create-app)
+3. A web application using your favorite CLI and language. We recommend:
+
+- [Vite for React projects](https://vitejs.dev/guide/#scaffolding-your-first-vite-project)
+- [Vue CLI](https://cli.vuejs.org/guide/creating-a-project.html#vue-create)
+- [Angular CLI](https://angular.io/cli/new)
+
+Monerium offers several authentication methods for web apps. The `MoneriumPack` implements the _Authorization code flow with proof key for code exchange (PKCE)_. Learn more about it [here](https://monerium.dev/docs/api#authentication).
+
+### Install dependencies
+
+```bash
+yarn add @safe-global/onramp-kit @monerium/sdk
+```
+
+### Login with Monerium
+
+Creating a _Login with Monerium_ integration for the Safe requires a multi-step process that should be implemented in your web app. The following steps are required.
+
+1. Load the application and instantiate the `OnRampKit` with the `MoneriumPack` using the following snippet:
+
+```typescript
+const onRampKit = await SafeOnRampKit.init(
```

Comment:
```suggestion

      const safeSdk = await Safe.create({
        ethAdapter: ethAdapter,
        safeAddress: selectedSafe,
        isL1SafeMasterCopy: true
      })

const onRampKit = await SafeOnRampKit.init(
```

Reply:
Not added because is exaplained in the next paragraph. If we want to write the complete snippet we need to add as well how to instantiate the EthersAdapter or Web3Adapter

Reply:
yep I think that makes sense to only reference the instantiation of the `SafeOnRampKit` with the `MoneriumPack`